martin/src/pg/config.rs

```diff
@@ -110,8 +114,18 @@ impl PgConfig {
 
     pub async fn resolve(&mut self, id_resolver: IdResolver) -> crate::Result<Sources> {
         let pg = PgBuilder::new(self, id_resolver).await?;
+
+        let instantiate_tables = pg.instantiate_tables();
+        pin_mut!(instantiate_tables);
```

Comment:
I think this should go inside the `fn instantiate_tables()` because otherwise you are not simultaneously instantiating tables AND functions. Instead, you instantiate tables for 5 seconds or until complet, and only then start instantiating functions.

Reply:
Sorry for the mixup there! This updated version should work as expected, but I wrapped the entire `instantiate_tables` function because there are multiple futures awaited inside of it and I wasn't sure the best way to go about that